src/pint/models/model_builder.py

```diff
@@ -1,3 +1,5 @@
+"""Building a timing model from a par file."""
```

Comment:
Any functional changes here or just sourcery?

Reply:
`ModelBuilder.__call__`, `get_model`, and `get_model_and_toas` functions now have an `allow_tcb` option. When this option is True, it will not throw an error when it encounters a TCB par file and silently reads the par file. The default behavior remains to throw an error for TCB par files.

Reply:
`ModelBuilder.__call__` will emit a warning about this,

Reply:
Wait, if you set `allow_tcb` you get a wrong model? Shouldn't it do the conversion?